src/libraries/Microsoft.PowerFx.Core/Public/Values/ErrorValue.cs

```diff
@@ -77,6 +78,12 @@ public override void ToExpression(StringBuilder sb, FormulaValueSerializerSettin
         {            
             var flag = true;
 
+            if (settings.Compact && _errors.Count == 1)
+            {
+                sb.Append(this);
```

Comment:
>sb.Append(this); [](http://example.com/codeflow?start=16&length=16)

This will call this.ToString(), which may not be the right thing. 

Does error need to behave differently with Compact settings?